posthog/models/filters/retention_filter.py

```diff
@@ -46,6 +48,8 @@ class RetentionFilter(
     PropertyMixin,
     DisplayDerivedMixin,
     FilterTestAccountsMixin,
+    BreakdownMixin,
+    BreakdownTypeMixin,
```

Comment:
Adding unneeded mixins on this filter so it's compatible with the rest of our event query classes. Follow up refactor is adjusting how our filter classes are split